tic-tac-toe.js

```diff
@@ -1,7 +1,77 @@
-function TicTacToe() {
+var TicTacToe = function() {
+ // initialize things when a new game starts
+ this.board = ["one", "two", "three", "four", "five", "six", "seven", "eight", "nine"];
+ this.player1 = true;
+ this.player2 = false;
+ this.score = [[9, 9, 9], [9, 9, 9], [9, 9, 9]];
+};
 
-}
 
-TicTacToe.prototype = {
-  
-}
+  TicTacToe.prototype.play = function() {
+//player 1 will be 1s and player 2 will be 2s.
+    var self = this;
+      self.board.forEach(function(cell) {
+        $("#" + cell).click(function() {
+          $("#" + cell).html(self.markBoard()); self.scoreGame($(this)); self.changePlayer();
+        });
+      }); //ends forEach
+    };  // ends play function
+
+//This function is called in .play
+  TicTacToe.prototype.markBoard = function() {
+    if (this.player1) {
+      return "<h1 class='x'>X</h1>";
+    } else {
+      return "<h1 class='o'>O</h1>";
```

Comment:
One thing I see here is that the difference between what is being returned is the X or the O. I think you could set a variable and then return the <h1> info only once. (This reduces the amount of duplicate code)